### DISCUSSION TITLE
About Apple In-App Purchase in wails

### DISCUSSION THREAD

**Discussion by B1ackg0d** [wailsapp/wails#3613] [General]
Hello everyone, is there an open-source solution for integrating Apple In-App Purchase into an app built on Wails? Any suggestions or solutions would be greatly appreciated.

**Comment by xudaolong** [comment]
i can't seem to find the information about this.

**Comment by likeablegeek** [comment]
I'm also curious about the same question. Is there a recommended approach to enabling IAP for macOS apps built in Wails. I've seen things like [go-iap](https://pkg.go.dev/github.com/awa/go-iap) but it isn't entirely clear to me if this will work or how to integrate it successfully in my Wails app.
